pkgs/top-level/all-packages.nix

```diff
@@ -20432,6 +20432,8 @@ with pkgs;
 
   collada-dom = callPackage ../development/libraries/collada-dom { };
 
+  coloquinte = callPackage ../development/libraries/science/electronics/coloquinte { };
```

Comment:
The letter `l` comes after `g`, please can you add it after `cog` ?

Reply:
Please ignore this comment, that file is totally messed up anyway.

Reply:
I did my best to minimize the number of mis-sorted packages :D cog seems very out-of-place

Reply:
yes indeed, sorry for the request change, you can ignore it.